components/View/FixedTabs/index.js

```diff
@@ -11,13 +11,15 @@ const propTypes = {
     }),
     onClick: PropTypes.func,
     active: PropTypes.string,
+    children: PropTypes.node,
 };
 
 const defaultProps = {
     active: undefined,
     className: '',
     tabs: [],
     onClick: () => {},
+    children: null,
 };
 
 export default class ScrollTabs extends React.PureComponent {
```

Comment:
Missed the proper class name (ScrollTabs -> FixedTabs)

Reply:
I do not understand this statement.